validation/policies/io/konveyor/forklift/vmware/changed_block_tracking_per_disk.rego

```diff
@@ -0,0 +1,25 @@
+package io.konveyor.forklift.vmware
+import future.keywords.in
+
+change_tracking_disabled_per_disk(disk) {
+    disk.changeTrackingEnabled == false
+}
+
+concerns[flag] {
+    some disk in input.disks
+    change_tracking_disabled_per_disk(disk)
+
+    path_parts := split(disk.file, "/")
+    filename := trim(path_parts[count(path_parts) - 1], "] ")
+
+    baseKey := (disk.controllerKey / 100) * 100
+    controllerIndex := disk.controllerKey - baseKey
+
+    deviceKey := sprintf("%s%d:%d", [disk.bus, controllerIndex, disk.unitNumber])
+
+    flag := {
+        "category": "Warning",
+        "label": sprintf("Disk - %s (key: %d, file: %s, datastore: %s) does not have CBT enabled", [deviceKey, disk.key, filename, disk.datastore.id]),
```

Comment:
Please remove the `(key: %d, file: %s, datastore: %s)` as it's not nessasry information